helm_deploy/values-preprod.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 4
 
 image:
-  repository: quay.io/hmpps/prisonstaffhub
+  repository: quay.io/hmpps/digital-prison-services
   tag: latest
```

Comment:
I have not changed the name of the cert_secret2 property in this file
cert_secret2: prison-staff-hub-cert
As I assume that this is pulled from somewhere with this name

Reply:
yes the cert is created in the cloudplatform environment repo https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/digital-prison-services-dev/07-certificates.yaml

Reply:
I've now changed this as I think we need host2 still - I'll raise a pull request on cloud-platform-environments